workease.rb

```diff
@@ -244,7 +247,7 @@ def warn(reason)
       @warn_log << message
     elsif Time.now.to_i > @pause_until
       `paplay ./sounds/when.ogg`
-      @pause_until = Time.now.to_i + 5
+      @pause_until = Time.now.to_i + 2
```

Comment:
Why was this changed?

Reply:
This should be 4 seconds, the warn method sleeps for 1 second, and displays the message for 3 seconds. If @pause_until is set to 4 seconds, you can click close on the message without it firing another warning straight away.